docs/connector-development/tutorials/custom-python-connector/1-environment-setup.md

```diff
@@ -14,13 +14,23 @@ cd airbyte-integrations/connector-templates/generator
 ./generate.sh
 ```
 
-Select Python-source Set name to survey-monkey-demo
+Select Python CDK Source Set name to survey-monkey-demo
 
-Next change your working directory to the new connector module. Then create an initial python
-environment and install the dependencies required to run an API Source connector:
+Next change your working directory to the new connector module. Also change airbyte-cdk version
+to the one used for this tutorial in `pyproject.toml`:
 
 ```bash
 cd ../../connectors/source-survey-monkey-demo
+```
+
+```toml
+airbyte-cdk = "0.78.1"
```

Comment:
Huh, is this because the new version breaks _the template_, or _the tutorial_? Either way, I would rather fix the tutorial and template than to tell people to pin to previous version.

Reply:
Ok, let me try to revisit the section to know if I can fix the problem and yes latest version breaks the tutorial.

Reply:
Well, I was finally able to revisit this, the concurrent section of the tutorial now works and I can remove this pin.